src/test/java/htsjdk/samtools/util/HttpUtilsTest.java

```diff
@@ -0,0 +1,33 @@
+package htsjdk.samtools.util;
+import java.io.IOException;
+import java.net.URL;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import htsjdk.HtsjdkTest;
+
+public class HttpUtilsTest extends HtsjdkTest {
+    @DataProvider(name = "existing_urls")
+    public Object[][] testExistingURLsData() {
+        return new Object[][]{
+            {"http://broadinstitute.github.io/picard/testdata/index_test.bam"},
+            {"http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/current.tree"}
+        };
+    }
+
+    @Test(dataProvider="existing_urls")
+    public void testGetHeaderField(final String url) throws IOException {
+        final String field = HttpUtils.getHeaderField(new URL(url), "Content-Length");
```

Comment:
I don't think that it will test the right thing if we use HTTPHelper....

Reply:
Since this test is testing the call to getHeaderField it seems like it would be problematic to replace it with a different method.  You could certainly make a case for merging the two methods but I'm not sure that's necessarily in the scope of this.